CMakeLists.txt

```diff
@@ -304,6 +206,9 @@ endif()
 # include as the last subdir
 add_subdirectory( vital )
 
+# Generate files to inform msvc of the kwiver runtime environment
+kwiver_setup_msvc_env() # Setup .bat scripts need to be complete and on disk
```

Comment:
Should this be conditional on MSVC build?

Reply:
Yeah, this should be wrapped